src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -1,43 +1,139 @@
 package cromwell.engine
 
-import akka.actor.Actor
+import akka.actor.{ActorRef, Props, Actor}
 import akka.actor.SupervisorStrategy.Stop
+import akka.event.LoggingReceive
+import cromwell.binding.{FullyQualifiedName, Workflow, WdlBinding, WdlValue}
 
 /**
  * This model assumes workflow actors are transient, created once per workflow instance.
  */
 object WorkflowActor {
```

Comment:
What's the advantage of that?  You could still send messages that were not WorkflowActorMessages to a WorkflowActor, would this let you pattern match to know that the actor was receiving something completely wrong?